install/tests/manifests/gcp-issuer.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: gitpod-issuer
+spec:
+  acme:
+    email: nandaja.varma@gmail.com
```

Comment:
As far as I know, you can simply remove the email line. This is only needed to notify when a cert expires (which is not relevant for test setups).

Reply:
If you need an email address for internal use as a sink for all emails from LetsEncrypt - we already use `letsencrypt-throwaway@gitpod.io` for that. With that being said - it it's possible to not specify an email at all then that is the better solution.

Reply:
turns out, I could just leave out the `email` key!